Tasks/VsTest/vstest.ts

```diff
@@ -73,13 +73,55 @@ catch (error) {
     throw error;
 }
 
-function getTestAssemblies(): Set<string> {
+function getBasePathFromFilter(testAssemblyFilter) {
```

Comment:
yes og t thissue, seemingly that will be because of the old logic. Will modify it.